traptor/traptor.py

```diff
@@ -155,6 +157,19 @@ def _setup_kafka(self):
             self.logger.debug('Kafka_enabled setting: {}'.format(self.kafka_enabled))
             self.kafka_conn = None
 
+    def _gen_kafka_success(self):
+        def kafka_success(tweet, response):
+            self.logger.info("Tweet sent to kafka: {}".format(tweet.get('id_str', None)))
+            dd_monitoring.increment('tweet_to_kafka_success')
```

Comment:
We should put the `id_str` into the `extras`, can't do aggregations on the message string